lib/phoenix_live_controller.ex

```diff
@@ -637,10 +630,18 @@ defmodule Phoenix.LiveController do
       def message_handler(socket, name, message),
         do: apply(__MODULE__, name, [socket, message])
 
+      def view_module,
+        do:
+          __MODULE__
+          |> to_string()
+          |> String.replace(~r/(Live|LiveController|Controller)?$/, "View", global: false)
+          |> String.to_existing_atom()
```

Comment:
I guess that with this, a cryptic `1st argument: not an already existing atom` error will be produced when view is yet to be added - which is a common case when you add a new controller via trial and error. I'd definitely prefer an error that explicitly states that module MyAppWeb.SomeView is missing, like regular Phoenix controller does.